detect_color_scheme.js

```diff
@@ -0,0 +1,16 @@
+var theme="light";    //default to light
+
+// local storage is used to override OS theme settings
+if(localStorage.getItem("theme")){
+    if(localStorage.getItem("theme") == "dark"){
+        var theme = "dark";
+    }
+} else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
+    // OS theme setting detected as dark
+    var theme = "dark";
+}
+
+//dark theme preferred, set document with a `data-theme` attribute
+if (theme=="dark") {
+     document.documentElement.setAttribute("data-theme", "dark");
```

Comment:
This will only pick up the setting the first time the page is loaded.  If I switch the system setting from light to dark, the docs will still be light.

Maybe the easiest way would be to just integrate https://github.com/GoogleChromeLabs/dark-mode-toggle, which seems to handle this case quite robustly.

Reply:
My newest commit should fix this, but I haven't had a chance to test it.

Reply:
> This will only pick up the setting the first time the page is loaded. If I switch the system setting from light to dark, the docs will still be light.
> 
> Maybe the easiest way would be to just integrate https://github.com/GoogleChromeLabs/dark-mode-toggle, which seems to handle this case quite robustly.

Does the newest commit solve this for you?